commit 7fada342ac3acc90752f21b31563e280ff7d2597 (refs/changes/40/3140/1)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2017-12-27T15:18:32-08:00 (1 year, 9 months ago)
    
    MANTA-3530 Update libmanta's node-moray version to smooth connection shuffling.

diff --git a/package.json b/package.json
index 82e16e9..1b51874 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "bunyan": "0.22.1",
         "jsprim": "^1.3.1",
         "lru-cache": "2.3.1",
-        "moray": "3.3.0",
+        "moray": "3.4.1",
         "once": "1.3.0",
         "restify": "2.6.3",
         "redis": "0.10.1",
